lib/src/main/java/com/metaplex/lib/drivers/storage/Storage.kt

```diff
@@ -1,15 +1,22 @@
 package com.metaplex.lib.drivers.storage
 
+import com.metaplex.lib.ASYNC_CALLBACK_DEPRECATION_MESSAGE
 import com.metaplex.lib.shared.ResultWithCustomError
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.suspendCancellableCoroutine
 import okhttp3.*
 import java.io.IOException
 import java.net.URL
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
 
 sealed class StorageDriverError: Exception() {
     object InvalidURL: StorageDriverError()
-    class CanNotParse(val error: Exception): StorageDriverError()
+    class CanNotParse(val error: Throwable): StorageDriverError()
```

Comment:
instead of Throwable? I think that would also work. I just went with the most generic type that would still work with everything, since Exception extends from Throwable.